packages/boot/test/orchestration/restart-contracts.test.ts

```diff
@@ -164,17 +174,34 @@ test.serial('stakeAtom', async t => {
     },
     proposal: {},
   });
-  // cosmos1test is from ibc/mocks.js
-  const accountPath = 'published.stakeAtom.accounts.cosmos1test';
+
   t.throws(() => readLatest(accountPath));
-  t.is(await flushInboundQueue(), 1);
-  t.deepEqual(readLatest(accountPath), {
-    localAddress:
-      '/ibc-port/icacontroller-1/ordered/{"version":"ics27-1","controllerConnectionId":"connection-8","hostConnectionId":"connection-649","address":"cosmos1test","encoding":"proto3","txType":"sdk_multi_msg"}/ibc-channel/channel-1',
-    remoteAddress:
-      '/ibc-hop/connection-8/ibc-port/icahost/ordered/{"version":"ics27-1","controllerConnectionId":"connection-8","hostConnectionId":"connection-649","address":"cosmos1test","encoding":"proto3","txType":"sdk_multi_msg"}/ibc-channel/channel-1',
-  });
-  // request-account is complete
+  t.throws(() => readLatest(accountPath2));
+
+  t.is(getInboundQueueLength(), 1);
+  await flushInboundQueue(1);
+
+  // The path changes depending on whether `send-anywhere` test runs previously.
+  const accountPath = 'published.stakeAtom.accounts.cosmos1test';
+  const accountPath2 = 'published.stakeAtom.accounts.cosmos1test1';
```

Comment:
how about querying the children of `published.stakeAtom.accounts` and choosing the address based on the results?